### PR TITLE
Leaderboard final

### PR DESCRIPTION
**Pull request by Nyame-Wolf**
In tihis pull request:
- I added styles to the leaderboard app

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,14 +14,34 @@
 
 ## Live Demo
 
-[Live Demo Link]()
+[Live Demo Link](https://nyame-wolf.github.io/Leaderboard/)
```
Comment (zieeco) [inline README.md:17]: - Please, kindly ensure the live demo link takes the user to the live version of your leaderboard app. You can choose to update the link or remove it to improve the user experience and to make your readme more professional.

Comment (zieeco) [reply README.md:17]: - Please, kindly add the "Acknowledgement" section and add it's contents to make your readme more professional.

File: README.md [hunk 2]
```diff
@@ -14,14 +14,34 @@
...
 
 ## Project Milestones
 
 - **Milestone 0 - project setup (0.5h)** ✅
 - **Milestone 1 - Leaderboard: hit the API (2.5h)**✅
 - **Milestone 2 - Leaderboard: final touches (4h)**
```
Comment (zieeco) [inline README.md:23]: - Please, kindly remove this added milestone as this is not required you add it, instead rename the "Project Milestones" to "Getting Started" as this is the correct header that should have all you've added in that section to make your readme more professional following the microverse readme template.

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @Nyame-Wolf ,


Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 3]
```diff
@@ -14,14 +14,34 @@
...
+
+### Install
+
+Run npm install to install packages
+
+### Usage
+
+Run npm run build
+run npm start
+
```
Comment (ahmetbozaci) [inline README.md:44]: - [ ] Like the last code reviewer said kindly add the acknowledgment section and fill it the proper way. [Requirement](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md#:~:text=Add%20descriptive%20README%20file%20to%20your%20project%20%2D%20please%20use%20this%20template%20but%20remember%20to%20customize%20it%20to%20your%20project.)

**Review by ahmetbozaci — CHANGES_REQUESTED** [review]
# ♻️♻️♻️ STATUS: Changes Required  ♻️♻️♻️ 


@Nyame-Wolf  ✋🏼 
#### Great work on making the changes requested by a previous reviewer 👏 
#### You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
___ 

### 💯  Highlights 💯
- Your git-flow is correct. ✔️
- You set up linter files and no errors. ✔️
- Your pull request is very professional. It has a meaningful title and summary. Keep it up.✔️
___

### ♻️ Request changes ♻️

- [ ] _Check the comments under the review._

___

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
___

### Cheers and Happy coding! 💻 👏
almost

___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._
___

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_

[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

**Review by Lameck1 — COMMENTED** [review]
Hello @Nyame-Wolf 👋,

_This is an additional review as a result of the second opinion request._

For this project, unlike some, adding the acknowledgment section in the README is not a hard requirement. It should have come as an OPTIONAL suggestion.

As this was the only reason that your project was not approved it will be approved as soon as possible.

Please ask for another review in your Dashboard and the next Code Reviewer will mark it as approved right away :)

Keep rocking 🚀!

**Review by topeogunleye — APPROVED** [review]
### Hi @Nyame-Wolf, 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations
>
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Followed Github flow :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good description :heavy_check_mark:
 
> ## [Optional] suggestions:
> * Nothing to mention.
> 
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**